File: Conditionals/timeday.py
# Tatiana susov, Time of Day, Python
import time

current = time.time()


local_time = time.localtime(current)
hour = local_time.tm_hour
print("It is", hour)

if hour >= 00 and hour < 12:
    print("Good morning!")
elif hour >=12 and hour <19:
    print("Good afternoon!")
else:
    print("Good evening!")

File: Conditionals/timeday.c
//Tatiana Susov, greet with time of day C
#include <stdio.h>
#include <time.h>

int main(void){
 
  time_t now = time(NULL);
     struct tm * tm_struct = localtime(&now);
     int hour = tm_struct->tm_hour;
     printf("%d", hour);

     if(hour >= 0 && hour < 12){
        printf("\nGood morning!");
     }else if (hour >= 16 && hour <= 18){
        printf("Good afternoon!");
     }else{
        printf("Good evening!");
     }
    return 0;
}